audio/music/xy/gymleaderbattle.asm

```diff
@@ -2853,102 +2853,6 @@ Music_GymLeaderBattleXY_Ch4_loop:
 	drum_note 7, 2
 	drum_note 12, 1
 	drum_note 4, 1
-	drum_note 7, 2
```

Comment:
This repetitive pattern just went on for 8 measures longer than it should have. This is the only debatably subjective fix imo.